internal/core/src/mmap/ChunkedColumn.h

```diff
@@ -347,27 +347,17 @@ class ChunkedVariableColumn : public ChunkedColumnBase {
                 ErrorCode::Unsupported,
                 "RawJsonAt only supported for ChunkedVariableColumn<Json>");
         }
-        if (offsets == nullptr) {
-            auto ca = SemiInlineGet(slot_->PinAllCells());
-            for (int64_t i = 0; i < num_rows_; i++) {
-                auto [chunk_id, offset_in_chunk] = GetChunkIDByOffset(i);
-                auto chunk = ca->get_cell_of(chunk_id);
-                auto valid = nullable_ ? chunk->isValid(offset_in_chunk) : true;
-                auto str_view = static_cast<StringChunk*>(chunk)->operator[](
-                    offset_in_chunk);
-                fn(Json(str_view.data(), str_view.size()), i, valid);
-            }
-        } else {
-            auto [cids, offsets_in_chunk] = ToChunkIdAndOffset(offsets, count);
-            auto ca = SemiInlineGet(slot_->PinCells(cids));
-            for (int64_t i = 0; i < count; i++) {
-                auto chunk = ca->get_cell_of(cids[i]);
-                auto valid =
-                    nullable_ ? chunk->isValid(offsets_in_chunk[i]) : true;
-                auto str_view = static_cast<StringChunk*>(chunk)->operator[](
-                    offsets_in_chunk[i]);
-                fn(Json(str_view.data(), str_view.size()), i, valid);
-            }
+        if (count == 0) {
```

Comment:
please also remove the default value of `offsets` and `count` of `BulkRawJsonAt` in header file `ChunkedColumnInterface`

Reply:
I think we can do the same for `BulkIsValid`?

`BulkRawStringAt` still needs `offsets = nullptr` for `CreateTextIndex`